vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -30,7 +30,7 @@ $global:excludeHashComparisionListInAzureChinaCloud = @(
 
 # defaultContainerdPackageUrl refers to the stable containerd package used to pull and cache container images
 # Add cache for another containerd version which is not installed by default
-$global:defaultContainerdPackageUrl = "https://acs-mirror.azureedge.net/containerd/windows/v1.6.21-azure.1/binaries/containerd-v1.6.21-azure.1-windows-amd64.tar.gz"
+$global:defaultContainerdPackageUrl = "https://acs-mirror.azureedge.net/containerd/windows/v1.6.35-azure.1/binaries/containerd-v1.6.35-azure.1-windows-amd64.tar.gz"
```

Comment:
I found these part also contains containerd versions
- https://github.com/Azure/AgentBaker/blob/390ee1f5a8e913ba2bf489e22dbb3a3c1247db12/parts/windows/windowscsehelper.ps1#L151-L154
- https://github.com/Azure/AgentBaker/blob/166558d2f3f3bce58c64fe68d29c0c11495ed33a/parts/windows/windowscsehelper.tests.ps1#L24-L40

Do we need to update? I also forgot them when I updated containerd 1.7

Reply:
It is the default containerd version installed by default in the VHDs to cache images. Containerd will be always installed in provisioning nodes.

Reply:
I think this part of code is deprecated now since we set the containerd package url in aks-rp code now. You could see the  comment. cc @AbelHu